IndomitableIbises/Bangazon-API-Sprint-1
# Computer model, view and url done.  Can post, put and delete.

Link to Ticket
Closes #8 

Description of Proposed Changes
Added the Department model, view and URL

Steps to Test
git fetch --all
git checkout JCcomputermodel
Delete existing database
python manage.py migrate
code .
look at the code and see if you have questions
python manage.py runserver
You should see three clickable urls on our page. Click on "http://localhost:8000/computers/" 
VERY IMPORTANT- I made the decom date just an integer field becuase I felt like it was better to just throw an expected date or whatever in there...tell me if that needs to change
Impacted Areas in Application


*Models, views, and urls are impacted by this code

Definition of Done
The project must be fully documented. This includes the following:
Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
Every class must be documented with purpose, author, and methods.
Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
The project must be able to run fully, and without errors, on each teammate's system.
Fulfills every requirement.
Every line of code has been peer reviewed.
For projects that require unit testing, core functionality must be identified and have at least one test for each
